FABulous/fabric_generator/fabric_gen.py

```diff
@@ -292,12 +303,14 @@ def generateConfigMemInit(self, file: str, globalConfigBitsCounter: int) -> None
                 writer.writerow(entry)
 
     def generateConfigMem(self, tile: Tile, configMemCsv: str) -> None:
-        """
-        This function will generate the RTL code for configuration memory of the given tile. It the given configMemCsv file does not exist, it will be created using `generateConfigMemInit`.
-
-        Args:
-            tile (Tile): A tile object
-            configMemCsv (str): The directory of the config memory csv file
+        """This function will generate the RTL code for configuration memory of the given tile. It the given configMemCsv file does not exist, it will be created using `generateConfigMemInit`.
```

Comment:
```suggestion
        """This function will generate the RTL code for configuration memory of the given tile. If the given configMemCsv file does not exist, it will be created using `generateConfigMemInit`.
```